tasks/installation.yml

```diff
@@ -1,15 +1,6 @@
 ---
-- name: update apt cache
```

Comment:
Is the cache automatically updated on Debian systems when we use the package module?

Reply:
[I don't think so.](https://stackoverflow.com/a/51547525) However, there is no need to update the cache at this point, because [package sources](https://wiki.debian.org/SourcesList#Example_sources.list) have not been modified previously. Or is `apache2` not included in Debian9 sources by default?

Reply:
apache2 should be included by default but I think it's better to have an updated cache. You probably want the latest version if you're not installing an explicit version

Reply:
Right, better be sure. I included the task again.